e2e/packages/sync-test/setup/openClientWithRootAccount.ts

```diff
@@ -7,5 +7,5 @@ dotenv.config({ path: "../contracts/.env" });
 export async function openClientWithRootAccount(page: Page, options?: { modeUrl?: string }) {
   const modeFragment = options?.modeUrl ? `&mode=${options.modeUrl}` : "";
   const privateKeyFragment = `&privateKey=${process.env.PRIVATE_KEY}`;
-  await page.goto(`http://localhost:3000?cache=false${modeFragment}${privateKeyFragment}`);
+  await page.goto(`http://127.0.0.1:3000?cache=false${modeFragment}${privateKeyFragment}`);
```

Comment:
looks like e2e tests are failing here, might revert this for now

```suggestion
  await page.goto(`http://localhost:3000?cache=false${modeFragment}${privateKeyFragment}`);
```